tests/test_mteb_results.py

```diff
@@ -62,17 +67,17 @@ def test_mteb_results():
         "dataset_revision": "1.0",
         "task_name": "dummy_task",
         "mteb_version": version("mteb"),
+        "evaluation_time": 100,
+        "kg_co2_emissions": None,
```

Comment:
Would ideally like to have evaluation time PR subset, but atm. we don't do that so this seems like the easiest way to do it.